### DISCUSSION TITLE
ticket page

### DISCUSSION THREAD

**Discussion by des-coder** [Peppermint-Lab/peppermint#315] [Ideas]
On the Ticket page a lot of information which is stored has been omitted, especially if a ticket was generated with the guest ticket URL;

Information such as the name and email. Could this be added?

![image](https://github.com/Peppermint-Lab/peppermint/assets/64350852/f52ed177-2c16-4ce9-8847-d537ed2d1ece)

Maybe here?

`Created by ${name} at ${date} | ${email}`

**Comment by potts99** [comment]
thank you! @des-coder

**Comment by potts99** [comment]
this has been sorted @des-coder
